programs/margin-account/src/lib.rs

```diff
@@ -53,9 +67,20 @@ pub struct Initialize<'info> {
 /// Initialize new margin collateral obligation.
 #[derive(Accounts)]
 pub struct InitObligation<'info> {
-    // TODO
-    #[account(signer)]
-    authority: AccountInfo<'info>,
+    lending_program: AccountInfo<'info>,
+    deposit_reserve: AccountInfo<'info>,
+    borrow_reserve: AccountInfo<'info>,
+    // ? This probably needs to be initialized
```

Comment:
this will be done client side

Reply:
ah yeah I meant to update the comments, will remove (and add the `mut` that's needed)